src/types/keyPair.ts

```diff
@@ -0,0 +1,6 @@
+import { ec as EC } from "elliptic";
+
+// tslint:disable-next-line: no-empty-interface
+export interface IKeyPair extends EC.KeyPair {
```

Comment:
I probably missed the reasoning behind this empty interface while swiftly checking this PR; could you enlighten the need for the empty interface?

Reply:
To expose the functions and properties of the parent interface: `EC.KeyPair` to the consumers. This was the quick and dirty way of achieving this.